changelog.txt

```diff
@@ -12,6 +12,7 @@
 * Tweak	- Display spinner icon during service data refresh.
 * Add	- Adds Dockerized E2E tests with GitHub Action integration.
 * Fix   - Handle DHL live rates notice creation and deletion errors.
+* Tweak - Update plugin author name.
```

Comment:
Does this need to go under `1.25.12`?

Reply:
Gonna blame Git's auto-merge for my inability to edit a text file correctly, but just moved it up into place now.